content/en/docs/concepts/cluster-administration/addons.md

```diff
@@ -21,7 +21,7 @@ Add-ons in each section are sorted alphabetically - the ordering does not imply
 
 
 * [ACI](https://www.github.com/noironetworks/aci-containers) provides integrated container networking and network security with Cisco ACI.
-* [Calico](https://docs.projectcalico.org/latest/getting-started/kubernetes/) is a secure L3 networking and network policy provider.
+* [Calico](https://docs.projectcalico.org/latest/introduction/) is a networking and network policy provider. Calico supports a flexible set of networking options so you can choose the most efficient option for your situation, including non-overlay and overlay networks, with or without BGP. Calico uses the same engine to enforce network policy for hosts, pods, and (if using Istio & Envoy) applications at the service mesh layer.
```

Comment:
Great description of Calico. I also verified that the new link is working properly.

Reply:
Also I believe this now addresses @sftim's concern. @sftim can you verify

Reply:
Thanks for checking. @bradtopol I'm happy with your opinion, and yes this looks fine to me.